app/code/Magento/Review/view/frontend/web/js/process-reviews.js

```diff
@@ -27,6 +29,14 @@ define([
                     event.preventDefault();
                 });
             });
+
+            if (hashAction) {
+                setTimeout(function () {
+                    $('html, body').animate({
+                        scrollTop: $('#' + hashAction).offset().top - 50
+                    }, 300);
+                }, 100);
+            }
         }).complete(function () {
             if (fromPages == true) { //eslint-disable-line eqeqeq
```

Comment:
Hi @niravkrish. I don't think we need to provide such workaround here. 
Take a look at the `if (fromPages == true)` condition. The problem is `fromPages` variable is always undefined. As result, the condition's body that scrolls the page to the correct position (reviews) is never being executed. So, we need to figure out why `fromPages` is always undefined and provide a fix.

Reply:
fromPages is already there before I code.
so what should i need to do for that.